next.config.js

```diff
@@ -3,17 +3,17 @@ const withPWA = require('next-pwa');
 const withImages = require('next-images');
 const withTM = require('next-transpile-modules')([
   '@patternfly/react-core',
-  '@patternfly/react-styles',
+  '@patternfly/react-styles'
 ]);
 
 module.exports = withPlugins([withTM, withImages, withPWA], {
```

Comment:
Since we'll be "deploying" the registry viewer under `/viewer`, I had to set `basePath: '/viewer',` in the config here, or else the paths for links got messed up with the proxy.

Might be prudent to also add a redirect for `/` to `/viewer`, but your call.

Reply:
Yeah... I dont know what to do here. The only use case for `/viewer` is for the registry and if someone else wants to deploy the viewer it doesnt really make sense.

Reply:
I agree it's a little weird, but Is there a case where we explicitly wouldn't want it to use /viewer? Is there an option to configure this?

The tricky part is that without the change, links within the viewer to other pages in the viewer get messed up

Reply:
Ill end up making an env var for this

Reply:
Fixed